msbuild/domoticz.vcxproj

```diff
@@ -29,27 +29,27 @@
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
     <CharacterSet>MultiByte</CharacterSet>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
Please undo these changes, better to undo all in this file
Also upgrade your visual studio ;)

Reply:
ok ! done ! i was on Winsows 7 so VS2019

Reply:
Windows 7 is deprecated as well as 8.1, better move to Windows 11, pretty all right once you get used to it (and stay of the inner circle development releases... 😁)